src/Admin/LoginPage.js

```diff
@@ -27,7 +27,7 @@ function LoginPage(props) {
 
     if (result.success) {
       console.log({ result });
```

Comment:
Remove `console.log` from git history only use console.log for testing and remove them before doing a commit